functions/graphql/index.js

```diff
@@ -73,7 +73,7 @@ function configureServer() {
       // CENTER
       // const token = "123456789012345678901234567890123456"; // Must be a 36 long string that matches the apiToken object on a center
       // DIRECTOR
-      // const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
+      const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```

Comment:
Make sure this is commented out so it doesn't get broken for me.
```suggestion
      // const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```